torchreid/engine/engine.py

```diff
@@ -778,6 +797,8 @@ def _feature_extraction(data_loader):
                 if self.use_gpu:
                     imgs = imgs.cuda()
 
+                # TODO(kprokofi) check features is tuple
```

Comment:
This branch isn't related to classification. if we ever get to check the functionality of this subtask, then we will fix what does not work

Reply:
Ok, I thought it will be useful in future. I'll remove it

Reply:
Done